tests/e2e/specs/in-game/game-over/rematch.spec.js

```diff
@@ -96,8 +96,7 @@ describe('Creating And Updating Ranked Matches With Rematch', () => {
     cy.setupGameAsP0(true, true);
   });
 
-  it('Wins match played with Rematch/Continue Match button', function () {
-    cy.skipOnGameStateApi();
+  it.only('Wins match played with Rematch/Continue Match button', function () {
```

Comment:
missed a .only(). We should write an issue to make this a lint error so we can see it automatically.
```suggestion
  it('Wins match played with Rematch/Continue Match button', function () {
```

Reply:
https://github.com/cuttle-cards/cuttle/issues/1131